lib/generator.rb

```diff
@@ -35,13 +37,43 @@ def test_cases
     cases.call(data)
   end
 
+  def metadata_repository_missing_message
+    <<EOM
+
+'#{METADATA_REPOSITORY}' repository not found.
+Try running the command:
+  git clone https://github.com/exercism/#{METADATA_REPOSITORY}.git ../#{METADATA_REPOSITORY}
+
+EOM
```

Comment:
What do you think about using either the `~EOM` or at least the `-EOM` form of the heredoc?  

The squigly heredoc is relatively new, so may not be correct to use yet.